cmake/kokkos_compiler_id.cmake

```diff
@@ -19,6 +19,7 @@ MACRO(kokkos_internal_have_compiler_nvcc)
   ENDIF()
 ENDMACRO()
 
+IF(NOT Kokkos_ENABLE_COMPILE_AS_CMAKE_LANGUAGE)
 IF(Kokkos_ENABLE_CUDA)
   # find kokkos_launch_compiler
   FIND_PROGRAM(Kokkos_COMPILE_LAUNCHER
```

Comment:
Can you please adapt the indentation here?

Reply:
done